src/Raven.Server/ServerWide/ClusterCommandsVersionManager.cs

```diff
@@ -85,6 +86,8 @@ public static class ClusterCommandsVersionManager
             [nameof(AddRavenEtlCommand)] = Base40CommandsVersion,
             [nameof(AddSqlEtlCommand)] = Base40CommandsVersion,
             [nameof(AddElasticSearchEtlCommand)] = Base40CommandsVersion,
+            
+            [nameof(AddQueueSinkCommand)] = Base40CommandsVersion,
```

Comment:
1. We also need to handle Add / Delete, don't we?
2. This is a new command so we must not use `Base40CommandsVersion`. I see the last one is `54_002` so newer commands should get `54_003`

Reply:
I added `UpdateQueueSinkCommand`. If I am right for Delete we are using `DeleteOngoingTaskCommand`.

Also, I don't need an abstract class here as I implemented it, can you please check and confirm before I change it?